nncf/common/accuracy_aware_training/runner.py

```diff
@@ -453,7 +448,8 @@ def update_training_history(self, compression_rate, metric_value):
         self._compressed_training_history.append((compression_rate, accuracy_budget))
 
         if IMG_PACKAGES_AVAILABLE:
-            plt.figure()
+            plt.ioff()
```

Comment:
AA algo also spawned a lot of figures in an interactive mode, leading to bad UX. I fixed it.